src/components/NewOrder.js

```diff
@@ -86,7 +92,8 @@ class NewOrder extends Component {
             this.state.entries.map((entry, i) => dom.tr(
               { className: 'entries', key: i + 1 },
```

Comment:
no reason...i think i left it from numbering each line item. i'll take it out to be less confusing.